modules/distr-tracing-tempo-config-query-rbac.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr-tracing-tempo-configuring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-configuring-query-rbac_{context}"]
+= Configuring the query RBAC
+
+As an administrator, you can set up the query role-based access control (RBAC) to filter the span attributes for your users by the namespaces for which you granted them permissions.
```

Comment:
nit: RBAC should be followed by a noun. Usually it is written as _RBAC rules_ or _RBAC permissions_.

(FWIW, I just wrote some docs with just RBAC, and this is fresh because another reviewer just caught this issue in my release notes :smiley_cat: )

This is not a merge blocker, but if you can, it would good to have this resolved before your release date.

Reply:
Side note: I think that when you change the term to _query RBAC rules_, the definitive article _the_ sounds a little off. For example, you might want to update your title to _Configuring query RBAC rules_ (or permissions).

Reply:
> nit: RBAC should be followed by a noun. Usually it is written as RBAC rules or RBAC permissions.

:slightly_smiling_face: I'd like to disagree on this point. For example: https://www.ibm.com/think/topics/rbac